### PR TITLE
Added comment entity.

### PR DESCRIPTION
**Pull request by EduarfF**

### CODE REVIEW

File: src/main/java/app/model/Comment.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+    @ToString.Exclude
+    @EqualsAndHashCode.Exclude
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "user_id", nullable = false)
+    private User user;
+
+    private String text;
+
+    private LocalDateTime timeStamp;
```
Comment (VlasenkoYevhenii) [inline src/main/java/app/model/Comment.java:40]:
```suggestion
    @Column(nullable = false)
    private LocalDateTime timeStamp;
```

File: src/main/java/app/model/Comment.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    @JoinColumn(name = "task_id", nullable = false)
+    private Task task;
+
+    @ToString.Exclude
+    @EqualsAndHashCode.Exclude
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "user_id", nullable = false)
+    private User user;
+
+    private String text;
```
Comment (VlasenkoYevhenii) [inline src/main/java/app/model/Comment.java:38]:
```suggestion
    @Column(nullable = false)
    private String text;
```

File: src/main/java/app/model/Comment.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
+package app.model;
```
Comment (VlasenkoYevhenii) [inline src/main/java/app/model/Comment.java:1]: CI check is failed. Conduct mcp mb?

**Review by VlasenkoYevhenii — CHANGES_REQUESTED** [review]

File: src/main/java/app/model/Comment.java [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+    @ToString.Exclude
+    @EqualsAndHashCode.Exclude
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "task_id", nullable = false)
+    private Task task;
+
+    @ToString.Exclude
+    @EqualsAndHashCode.Exclude
+    @ManyToOne(fetch = FetchType.LAZY)
```
Comment (VlasenkoYevhenii) [inline src/main/java/app/model/Comment.java:33]:
```suggestion
    @ManyToOne
```

File: src/main/java/app/model/Comment.java [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+@Table(name = "comments")
+@Data
+public class Comment {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ToString.Exclude
+    @EqualsAndHashCode.Exclude
+    @ManyToOne(fetch = FetchType.LAZY)
```
Comment (VlasenkoYevhenii) [inline src/main/java/app/model/Comment.java:27]:
```suggestion
    @ManyToOne
```

**Review by VlasenkoYevhenii — CHANGES_REQUESTED** [review]

File: src/main/java/app/model/Comment.java (hunk 4 above)
Comment (bodya4243) [reply src/main/java/app/model/Comment.java:33]: as far as i know it's Eager in default

**Pull request merged**